src/schema/artist/carousel.js

```diff
@@ -38,7 +38,7 @@ const ArtistCarousel = {
         published: true,
       }),
     ])
-      .then(([shows, artworks]) => {
+      .then(([{ body: shows }, artworks]) => {
```

Comment:
We wound up changing the signature of this loader to resolve `{body, headers}`, instead of `body`, since when we use that loader for a connection, we need to get the total count (which only comes back from Gravity, in this endpoint, in the header).

That's the reason for some of these changes sprinkled throughout. Any place that just uses `shows` directly needs to be destructured into `{body: shows}`